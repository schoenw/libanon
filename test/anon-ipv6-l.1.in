31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
5dc9:b9a:5699:45e4:3c33:2f50:a4d:2fca
2d70:77b8:119d:918:4165:78ae:2f24:1d00
17a3:638e:1942:258e:c87:4416:5298:4071
1c12:2019:e3f:648a:1689:65e9:1bcb:2436
2cc:5582:1702:485c:4a77:5077:575c:1035
bba:3c9d:5b3a:21de:2cf8:6578:2ae7:779f
e22:869:6f4f:55f9:73d2:7795:3881:1671
827:7f95:2fe6:531f:d63:7bdf:323f:3ef6
1e45:45f7:d60:36d0:1827:6b55:5dae:5741
31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
1e45:45f7:d60:36d0:1827:6b55:5dae:5741
31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
31d6:1be5:41bd:663c:3e9e:1005:1eec:6f01
fffe:638:709:861::3
fffe:638:709:861::4
ffff:638:709:861::1
ffff:638:709:861::2
2001:638:709:861::1
2001:638:709:861::2
2001:638:709:861::3
2001:638:709:861::4
2001:638:709:4::1
2001:638:709:4::2
2001:638:709:4::3
2001:638:709:4::4
