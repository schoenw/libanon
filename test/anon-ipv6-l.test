#!/bin/bash
#
# Shell script for regression testing libanon (anon-ipv6-l).
#
# $Id$
#

ANON=../src/anon
PASSPHRASE=testing

RC=0
for file in anon-ipv6-l.*.in; do
    $ANON ipv6 -p $PASSPHRASE -l $file \
	| diff -u `basename $file .in`.out -
    if [ $? -ne 0 ]; then
 	RC=1
    fi
done

exit ${RC}