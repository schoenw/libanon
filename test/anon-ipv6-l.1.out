3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
63cf:3bda:5768:b87b:ac8c:cc56:a43:5235
2cce:77ba:d5ad:16df:c119:68b2:2f0a:2518
17a3:94a6:da41:21f1:8fa6:bbe9:aa59:c001
1df2:1602:ce3f:5a8a:5da9:597:e43b:1bce
6cc:4a6d:2f3d:775b:cb88:ae6f:a8a2:6fc1
bbd:c302:a5f5:d85e:3308:768:2570:fa5f
dad:da60:80b0:bc67:c2c:7b50:3873:e40f
927:7fea:c099:b4df:4e9d:73ac:2d3f:eb06
1efb:a377:29d:3a2d:e8a6:eb59:a391:d621
3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
1efb:a377:29d:3a2d:e8a6:eb59:a391:d621
3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
3c29:fb1a:b9fd:613c:e68:a03b:1efd:961f
8000:ce05:ee9:971:7f1f:1f8:ec7:c05b
8000:ce05:ee9:971:7f1f:1f8:ec7:c05f
8001:e9f8:3928:f76e:3f:3fc6:135:ffa4
8001:e9f8:3928:f76e:3f:3fc6:135:ffa7
20f1:f603:8e9:98c:3f5f:1f9:ffcf:bf99
20f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9a
20f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9b
20f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9c
20f1:f603:8e9:6cb:803f:3e39:f134:3fb8
20f1:f603:8e9:6cb:803f:3e39:f134:3fba
20f1:f603:8e9:6cb:803f:3e39:f134:3fbb
20f1:f603:8e9:6cb:803f:3e39:f134:3fbf
